src/app/page.tsx

```diff
@@ -5,7 +5,7 @@ import logoImage from "@/assets/logo.svg";
 import HeroSection from "@/components/HeroSection";
 import NewLGTMButton from "@/components/NewLGTMButton";
 
-export default function Home() {
+export default async function Home() {
```

Comment:
```suggestion
export default function Home() {
```

ローディング画面を試しで表示するのに加えたのですが、削除するのを忘れてました！修正します！